glue_astronomy/translators/regions.py

```diff
@@ -93,7 +105,10 @@ def roi_subset_state_to_region(subset_state, to_sky=False):
         raise NotImplementedError(f"ROIs of type {roi.__class__.__name__} are not yet supported")
 
     if to_sky:
-        reg = reg.to_sky(subset_state.xatt.parent.coords)
+        if override_wcs is not None:
+            reg = reg.to_sky(override_wcs)
+        else:
+            reg = reg.to_sky(subset_state.xatt.parent.coords)
```

Comment:
```suggestion
            reg = reg.to_sky(subset_state.xatt.parent.coords)
        elif to_sky:
            warnings.warn(f"Invalid input {to_sky} for ``to_sky``, "
                          "must be a valid WCS or set to `True`.",
                          UserWarning)
```
This is really for the change suggested by @pllim, but I cannot add suggestions on the suggestion...
But if we go that path, I think at that point (`not isinstance(to_sky, BaseHighLevelWCS)`) a sanity check is due, else it will appear to just silently ignore the `to_sky` input.